recipe/meta.yaml

```
@@ -35,12 +36,24 @@ requirements:
     - joblib >=0.14.1
 
 test:
+  source_files:
+    - tests
   imports:
     - phik
-  requires:
-    - pip
+    - phik.binning
+    - phik.data_quality
+    - phik.decorators
+    - phik.outliers
+    - phik.phik
+    - phik.significance
+    - phik.simcore
+    - phik.utils
   commands:
     - pip check
+    - pytest -v tests
+  requires:
+    - pip
+    - pytest
```

```suggestion
    - pytest >=4.0.2
```
Not a big deal but https://github.com/KaveIO/PhiK/blob/v0.12.4/pyproject.toml#L32